pathz/authorization-README.md

```diff
@@ -27,15 +27,13 @@ methodology (READ/WRITE). The result of an Authorization evaluation is an
 Action (PERMIT/DENY), policy version, and rule identifier.
 
 A Best, or most specific, match is that which has the longest match to the
-requested path and prefers:
+requested path and prefers in order:
 
-* a specific user over a group in the matching policy.
-* a defined KEY over a wildcard element in a keyed path.
-* a mode which matches that of the request (READ/WRITE).
-
-Authorization rules must be defined such that a single best match is possible.
-If the result of policy evaluation is more than one match, an error must be
-raised.
+1. Definite keys over wildcards keys. A rule with more definite keys is
+   preferred over one with fewer.
+2. User over group. A rule that matches with the user is preferred over one
+   with matches with a group a user belongs to.
+3. Deny over permit. If all above are equal, prefer the rule that with DENY action.
```

Comment:
'prefer the rule that with DENY action'

perhaps:
  'prefer the rule with DENY action"